scripts/process_new_format.py

```
@@ -107,6 +107,7 @@ class Ruling(BaseModel):
 
 
 def load_faqs(faqs_path: Path) -> dict[str, dict[str, str]]:
+
     # Path validation and loading logic
 def load_faqs(faqs_path: Path) -> dict[str, dict[str, str]]:
```

Ruff (0.11.9)

SyntaxError: Expected an indented block after function definition

---

Expected 2 blank lines, found 1

(E302)